### DISCUSSION TITLE
Chia coins are gone((((

### DISCUSSION THREAD

**Discussion by ANDRIY19861210** [Chia-Network/chia-blockchain#8194] [Support]
Yesterday I opened the Chia Blockchain and discovered that my 0.6 Chia coins had disappeared. The balance as of today is 0.02 Chia. There is no information about the original transactions in the transaction list. The wallet is synchronized. At the same time property on hpool. If this is a program error, how can I fix it? If it is a theft, how to confirm it?

**Comment by ANDRIY19861210** [comment]
From this purse was stolen my 0,56 chia coin - xch1knw4wjy64yv9ghjh9l2ahe83eldxg4a06x75dzwwem47qrkpj9vs2qe8cq.

**Comment by loppefaaret** [comment]
if another machine is used to do outgoing transactions, they will not be listed in the transactions list of other clients.
if you did not yourself make an outgoing transaction, im afraid someone else has gotten hold of your 24 word mnemonic key, and is transferring the funds out of the wallet!
either someone has tricked you into revealing it, or perhaps you have run 3rd party software that has stolen the key.
i would recommend that you create a new wallet / 24 word mnemonic, and transfer any remaining funds out to the new wallet.
try and identify how anyone else might have gotten a hold of your 24 words.
if you are unable to identify the method, i can only recommend a full OS reinstall.

**Comment by Jacek-ghub** [comment]
If stolen 24 word mnemonic (24WM) is a possibility, why not extend it to 24WM PLUS a password and generate a private key (so that private key PLUS password will be used to initialize other installations, not just that 24WM).  If any third part software will lift those mnemonics, they will choke for some time on password.  Or like SSH does it for ages, make the password optional, so no private key is needed.

Of course, if someone has a keylogger and a full box access all bets are off.

**Comment by BrandtH22** [comment]
@ANDRIY19861210 , unfortunately it appears someone has gained access to your keyset and/or device and stolen your funds. Since chia is a truly decentralized network there is no way to reclaim the funds or revert the transaction(s). One thing we recommend is ensuring that you and your farm are following security best practices to reduce malicious actors ability to gain access to your keys and/or device, many of these best practices are outlined here: https://www.chia.net/2021/05/28/securing-your-chia-how-to-be-a-hard-target/

Since we unfortunately cannot do more to assist with the stolen funds we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
